config/routes.rb

```diff
@@ -0,0 +1,22 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  get 'users/login_form'
+  get 'users/login'
+  get "/login", to: "users#login_form", as: "login"
+  post "/login", to: "users#login"
+  post "/logout", to: "users#logout", as: "logout"
+  get "/users/current", to: "users#current", as: "current_user"
+
+  get '/works/top', to: 'works#top'
+
+  resources :works do
+    member do
+      post :upvote
+    end
+  end
+
+  resources :works
```

Comment:
This is redundant because line 12 does the same thing. Having this repeated actually duplicates some routes, and makes it cluttered! We should be able to safely delete this! When I delete it, your tests all still pass.